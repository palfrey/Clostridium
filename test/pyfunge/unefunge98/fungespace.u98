>;"a#<"eno tsrif setirwrevo enil dnoces :DAB">:#,_@;      ;@_,#! #:<"BAD: second line is ignored"a0n"
 #                                                        ;'
>                                                         ;@,a.$_,#! #:<"BAD: Funge space contains unexpected character "0;:88*2--#;_0a"derongi si enilwen dna detanetacnoc era senil tneuqesbus :DOOG">:#,_@
