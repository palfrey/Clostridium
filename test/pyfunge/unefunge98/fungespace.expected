GOOD: subsequent lines are concatenated and newline is ignored

